.github/workflows/regression-tests/action.yml

```
@@ -0,0 +1,49 @@
+name: Common tasks
```

Maybe give this a name specific to regression tests e.g.
```suggestion
name: Regression Tests setup
```